src/Supportive/File/Dumper.php

```diff
@@ -30,7 +31,7 @@ public function dump(string $file): void
         if ($filesystem->exists($target->getPathname())) {
             throw FileAlreadyExistsException::alreadyExists($target);
         }
-        if (!$target->isWritable()) {
```

Comment:
FYI, a file that doesn't exist is never writable. Instead, we have to check the path (i.e. the directory containing the file), if it is writable, which is why this check never worked.